modules/GenericBase/sql/install.sql

```diff
@@ -0,0 +1,100 @@
+--
+-- Table structure for table `moh_import_data`
+--
+
+DROP TABLE IF EXISTS `moh_import_data`;
```

Comment:
move to importData module in the clinikal-backend repo

Reply:
ok